packages/@uppy/utils/src/delay.ts

```diff
@@ -2,12 +2,8 @@ import { createAbortError } from './AbortController.js'
 
 /**
  * Return a Promise that resolves after `ms` milliseconds.
- *
- * @param {number} ms - Number of milliseconds to wait.
- * @param {{ signal?: AbortSignal }} [opts] - An abort signal that can be used to cancel the delay early.
- * @returns {Promise<void>} A Promise that resolves after the given amount of `ms`.
```

Comment:
Generally I only like comments that elaborate on code, rather than explaining code.

When you read `opts: { signal: AbortSignal }` I think we all understand what that means and "An abort signal that can be used to cancel the delay early" is not adding much. Ditto for `ms`?